src/aiori.h

```diff
@@ -83,8 +84,14 @@ typedef struct ior_aiori {
         void (*initialize)(); /* called once per program before MPI is started */
         void (*finalize)(); /* called once per program after MPI is shutdown */
         option_help * (*get_options)();
+        bool enable_mdtest;
 } ior_aiori_t;
 
+enum bench_type {
```

Comment:
I find this solution a bit crude. AIORI at some point might become a library and useful to other benchmarks. But well, Ok for now.